app/views/entities/new.html.erb

```diff
@@ -0,0 +1,55 @@
+<nav class="navbar navbar-collapse navbar-dark bg-dark">
+  <div class="container-fluid">
+    <%= link_to groups_path, class: "navbar-brand" do %>
+      <%= image_tag "left-arrow.png" %>
+    <% end %>
+    <h2 class="navbar-brand mb-0">TRANSACTIONS</h2>
+    <div>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav ms-auto">
+          <li class="nav-item">
+            <%= link_to "Home", root_path, class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "About", root_path, class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "Contact", root_path, class: "nav-link" %>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</nav>
+<p class="notice ms-3 mt-3"><%= notice %></p>
+<p class="alert ms-3 mt-3"><%= alert %></p>
+<div class="container">
+  <%= form_with model: @entity, url: group_entities_path(@group), method: :post, local: true, html: { class: "needs-validation", novalidate: true } do |f| %>
+    <div class="mb-3 text-center">
+      <h2>ADD TRANSACTION</h2>
+    </div>
+    <div class="row g-3">
+      <div class="col-md-6">
+        <%= f.text_field :name, autocomplete: "Entity name", placeholder: "Name", class: "form-control", required: true %>
+      </div>
+      <div class="col-md-6">
+        <%= f.text_field :amount, autocomplete: "Amount", placeholder: "Amount", class: "form-control", required: true %>
+      </div>
+      <div class="col-md-12">
+        <% @groups.each do |group| %>
```

Comment:
### Add a new transaction page

- [ ]  According to the project requirements, When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category and not the Home Page. Kindly correct this as well. ✍🏻 

- [ ] It's one of the project requirements that The user fills out a form to create a new transaction or expense with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one).  kindly prevent the user to create a new transaction without selecting at least one category. You can make it an input field or selector...


![image](https://user-images.githubusercontent.com/79058364/236524400-e887e328-603b-4894-a87d-9d15a03c534a.png)